src/nbl/video/device_capabilities/device_features.json

```diff
@@ -1106,7 +1106,7 @@
         {
             "type": "bool",
             "name": "dynamicRendering",
-            "value": false,
+            "value": true,
             "expose": "DISABLE",
```

Comment:
you need to REQUIRE it, but don't do it yet because we're not implementing dynamic rendering yet